doc-content/enterprise-only/openshift/template-deploy-mandatory-proc.adoc

```diff
@@ -0,0 +1,107 @@
+[id='template-deploy-mandatory-{context}-proc']
+= Setting required parameters
```

Comment:
In this section and all similar section, you need to give context. Remember that each module must be able to make sense standalone.  Consider something along these lines:
The BLAH template does (WHATEVER). You must set the parameters listed in this procedure to do WHATEVER.

Reply:
I did add a brief formula (and a variable so it can work, too). But I'm afraid this will need further discussion. The formula is quite brief - "when configuring the template to deploy ...", but  whatever else I can come up with sounds incredibly awkward. Marked for restructure JIRA.